ui/src/api/loginBase.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Unauthenticated `/login/*` endpoints.
+ * Separate from authenticated endpoints to avoid import cycle with `api.js`.
+ */
+import baseApi from './apiBase';
+
+const endpoint = baseApi.url('/login');
+
+export function sendLogIn(proposal, password, previousUser) {
+  return endpoint.post({ proposal, password, previousUser }, '/').safeJson();
+}
+
+export function sendSSOLogIn() {
+  window.location = 'mxcube/api/v0.1/login/ssologin';
+}
+
+export function fetchLoginInfo() {
```

Comment:
Probably very few care about this, but anyway... Can we be more consistent with casing? : D

```suggestion
export function sendLogin(proposal, password, previousUser) {
  return endpoint.post({ proposal, password, previousUser }, '/').safeJson();
}

export function sendSsoLogin() {
  window.location = 'mxcube/api/v0.1/login/ssologin';
}

export function fetchLoginInfo() {
```

My rule of thumb is to think how I would write in snake case: `send_s_s_o_log_in` or `send_sso_login`.

Reply:
- It's the verb "to log in" in this case - i.e. send + command (with some exceptions to avoid duplicating the verb "send", like `sendFeedback`).
- Regarding casing of abbreviations, that's a whole other discussion. Even the web platform is inconsistent (e.g. [`XMLHttpRequest`](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) :rofl:) Personally, I think enforcing a specific case is unnecessary.
- Anyway, lucky for us, `sendSSOLogIn` will be removed with #1517 :joy: